src/alerts/check_signals_updates.R

```diff
@@ -0,0 +1,207 @@
+box::use(jsonlite[fromJSON, toJSON])
+box::use(httr[GET, POST, add_headers, status_code, content])
```

Comment:
A bit of a pain, but I think I am going to propose that we don't import objects like above but instead import the module and the reference exported objects using `module$object` notation. If the module is used frequently or has a long name, you could alias it like we do with `../utils/cloud_storage`.

Reply:
As well, recommend to shift away from `{httr}` which has been superseded by `{httr2}`. Will add some rough attempt to convert the functions (without testing) in your code below!

Reply:
Sounds good to me